### ISSUE TITLE
Conduct the card sort and synthesize findings

### ISSUE THREAD

**Issue by lauraponce** [usdoj-crt/beta-ada#42]
Dependent on #40 

Run the card sort sessions and synthesize the findings.

### This is done when:

- [ ] Card sort sessions are completed
- [ ] Research has been synthesized 
- [ ] Shareback session has been scheduled
- [ ] Findings are written up and shared back with the team for discussion
- [ ] New issues and future work has been added to GitHub
